docs/reference/controls/itemscontrol.md

```diff
@@ -89,7 +115,61 @@ public class Crockery
 
 The view resizes horizontally, but content is hidden when it is too high. This control does not have a built-in scrollbar (unlike `ListBox`).
 
-<img src={ItemsControlScreenshot} alt="" />
+<img src={example01Screenshot} alt="" />
+
+## Example 02
+This example bind an observable collection of tile items to an items control. The ItemPanel is set to a canvas and we use a style to position the Tile within the canvas.
+
+```xml
+<ItemsControl ItemsSource="{Binding TileList}">
+  <ItemsControl.ItemsPanel>
+    <ItemsPanelTemplate>
+      <Canvas Width="50" Height="50" Background="Yellow" Margin="3"/>
+  </ItemsPanelTemplate>
+  </ItemsControl.ItemsPanel>
+  <ItemsControl.ItemTemplate>
+    <DataTemplate>
+      <Rectangle Fill="Green" Height="{Binding Size}" Width="{Binding Size}"/>
+    </DataTemplate>
+  </ItemsControl.ItemTemplate>
+  <ItemsControl.Styles>
+    <Style Selector="ContentPresenter"  x:DataType="vm:Tile">
+      <Setter Property="Canvas.Left" Value="{Binding TopX}"/>
+      <Setter Property="Canvas.Top" Value="{Binding TopY}"/>
+    </Style>
+  </ItemsControl.Styles>
+</ItemsControl>
+```
+
+```csharp title='C# View Model'
+using AvaloniaControls.Models;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+
+namespace AvaloniaControls.ViewModels
+{
+    public class MainWindowViewModel : ViewModelBase
+    {
+        public ObservableCollection<Tile> TileList { get; set; }
+        
+        public MainWindowViewModel()
+        {
+            TileList = new ObservableCollection<Tile>(new List<Tile>
```

Comment:
I am not sure if this is a good idea putting this VM in the examples. It sort of _loosely_ breaks MVVM by making the VM directly care about presentation. But other people may say otherwise.

Reply:
I just did the same things as the already existing example in the page. Happy to change that. What would be better? (I'm happy to change the existing example too for consistency)

Reply:
I think in that case it is a valid option. Could be a representation of whatever. A pin mark on a map or similar. So I think this is okay, but probably some comments are a good idea.

Reply:
all right, what comment would be the best to add?

Reply:
for example: `// This is just some sample data storing the coordinates directly inside the item-model. In your real-world App you may want to use a converter or similar to calculate the position`